Scheduler/Notification.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Scheduler
+{
+	/// <summary>
+	/// Notification Table Info
+	/// </summary>
+	public class Notification
+	{
+		public long NotificationId { get; set; }
+
+		[ForeignKey("Task")]
+		public long? TaskId { get; set; }
+
+		public DateTime DateTimeStart { get; set; }
```

Comment:
```suggestion
		public DateTime NotificationStartTime { get; set; }
```

I think previous is like about creation date so it's better to highlight the purpose